test/runtest/syscfg.yml

```diff
@@ -19,11 +19,42 @@
 # Package: test/runtest
 
 syscfg.defs:
+    RUNTEST_LOG:
+        description: >
+            Causes test results to be logged to a cbmem-backed logged.
+        value: 0
```

Comment:
Maybe default this to 1? This might make more sense to be defaulted on. I'm fine either way :)